addons/interaction/CfgVehicles.hpp

```diff
@@ -1,25 +1,45 @@
+#define MACRO_PORTABLELIGHTSACTIONS(var1) \
+    class ACE_Actions { \
+        class ACE_MainActions { \
+            displayName = CSTRING(MainAction); \
+            selection = ""; \
+            distance = 2; \
+            condition = "true"; \
+            class ACE_SwitchLamp { \
+                displayName = CSTRING(var1); \
+                condition = QUOTE(alive _target); \
+                statement = QUOTE([ARR_1(_target)] call DFUNC(switchLamp)); \
+                selection = ""; \
+                distance = 2; \
+                showDisabled = 0; \
+                priority = -1; \
+            }; \
+        }; \
+    }; \
+    class ACE_SelfActions {}
+
 class CfgVehicles {
-  class ACE_Module;
-  class ACE_ModuleInteraction: ACE_Module {
-    author = ECSTRING(common,ACETeam);
-    category = "ACE";
-    displayName = CSTRING(Module_DisplayName);
-    function = "ACE_Interaction_fnc_moduleInteraction";
-    scope = 2;
-    isGlobal = 1;
-    icon = PATHTOF(UI\Icon_Module_Interaction_ca.paa);
-    class Arguments {
-      class EnableTeamManagement {
-        displayName = CSTRING(EnableTeamManagement_DisplayName);
-        description = CSTRING(EnableTeamManagement_Description);
-        typeName = "BOOL";
-        defaultValue = 1;
-      };
-    };
-    class ModuleDescription {
-      description = CSTRING(Module_Description);
+    class ACE_Module;
+    class ACE_ModuleInteraction: ACE_Module {
+        author = ECSTRING(common,ACETeam);
+        category = "ACE";
+        displayName = CSTRING(Module_DisplayName);
+        function = "ACE_Interaction_fnc_moduleInteraction";
```

Comment:
Seems like the best place to change, please change it to `QFUNC(moduleInteraction)`.